File: My Project/AndroidProject/MasterAnimation/app/src/main/java/com/theannguyen/masteranimation/MainActivity.java
package com.theannguyen.masteranimation;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView txtAnimation;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        txtAnimation = (TextView) findViewById(R.id.txtAnimation);

        txtAnimation.setTextAlignment(View.TEXT_ALIGNMENT_CENTER);
        txtAnimation.setTextSize(20);
        txtAnimation.setWidth(400);
        txtAnimation.setHeight(50);


        txtAnimation.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                txtAnimation.animate().rotationBy(100).translationYBy(100);
            }
        });

    }
}

File: Android_Kotlin/DemoKotlin/app/src/main/java/com/theannguyen/demokotlin/MainActivity.kt
package com.theannguyen.demokotlin

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        txtText.setText(R.string.text_value)
        btnClick.setOnClickListener {
//            txtText.text = "Ahii"
//            Toast.makeText(this,"Hello",Toast.LENGTH_SHORT).show()
            val intent = Intent(this,BeHocToan::class.java)
            startActivity(intent)
        }
//        btnClick.setOnClickListener(object : View.OnClickListener{
//            override fun onClick(v: View?) {
//                TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
//            }
//
//        })

    }
}
